datafusion/CHANGELOG.md

```diff
@@ -19,6 +19,340 @@
 
 # Changelog
 
+## [14.0.0](https://github.com/apache/arrow-datafusion/tree/14.0.0) (2022-11-04)
+
+[Full Changelog](https://github.com/apache/arrow-datafusion/compare/13.0.0-rc1...14.0.0)
```

Comment:
This link seems broken.

Reply:
Yes, this link won't exist until I tag the repo with `14.0.0` once the vote passes.